tests/src/JIT/Regression/JitBlue/CoreCLR_13910/CoreCLR_13910.cs

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Runtime.InteropServices;
+
+// Represents a problem with contained nodes chaines, that contain lclVar reads, that were moved through lclVar stores.
+// Notice that the project file sets complus_JitStressModeNames.
+
+[StructLayout(LayoutKind.Explicit)]
+internal class AA
+{
+
+    [FieldOffset(8)]
+    public QQ q;
+
+
+    public static AA[] a_init = new AA[101];
+
+    public static AA[] a_zero = new AA[101];
+
+
+    public AA(int qq)
+    {
+
+        this.q = new QQ(qq);
+    }
+
+
+    public static void reset()
+    {
+        AA.a_init[100] = new AA(1);
+        AA.a_zero[100] = new AA(2);
+    }
+}
+
+internal class QQ
+{
+    public int val;
+
+    public QQ(int vv)
+    {
+        this.val = vv;
+    }
+
+    public int ret_code()
+    {
+        return 100;
+    }
+}
+
+internal class TestApp
+{
+
+    private static int test_2_2(int num)
+    {
+        int result;
+        if (AA.a_init[num].q != AA.a_zero[num].q) 
+        // Access field with contained IND instruction.
+        // EQ marks its operands as contained too. 
+        // AA.a_init[num].q and AA.a_zero[num].q are allocated to the same lclVar.
+        // So we calculate AA.a_init[num].q and store as tmp0, use this temp to do nullCheck.
+        // Then store AA.a_zero[num].q as tmo0, destroy the old value and try to do EQ thinking that 
```

Comment:
tmo0 => tmp0

Reply:
Fixed

Reply:
It is not clear why `AA.a_init[num].q` and `AA.a_zero[num].q` would be allocated to the same lclVar; I presume that the JIT is doing some reuse of struct temps?

Reply:
> It is not clear why AA.a_init[num].q and AA.a_zero[num].q would be allocated to the same lclVar; I presume that the JIT is doing some reuse of struct temps?

Yes